src/app/courses/components/course/course.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="card">
-  <div class="content">
+<div class="card" [appFreshCourse]="course">
+  <div class="content" [ngClass]="course.topRated ? 'topRated': ''">
```

Comment:
It's better to use `[ngClass]="{ 'className': condition }"` to avoid  ? : operator with empty string.
( In your case: `[ngClass]="{ 'topRated': course.topRated }"` )